curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-89-roman-numerals.md

```diff
@@ -8,7 +8,18 @@ dashedName: problem-89-roman-numerals
 
 # --description--
 
-For a number written in Roman numerals to be considered valid there are basic rules which must be followed. Even though the rules allow some numbers to be expressed in more than one way there is always a "best" way of writing a particular number.
+For a number written in Roman numerals to be considered valid there are basic rules which must be followed. Even though the rules allow some numbers to be expressed in more than one way there is always a *best* way of writing a particular number.
+
+- Numerals must be arranged in descending order of size.
+- M, C, and X cannot be equalled or exceeded by smaller denominations.
```

Comment:
This is a non-American way of spelling.

```suggestion
- M, C, and X cannot be equaled or exceeded by smaller denominations.
```

Reply:
Looking around both seems to be used, one is preferred in the American-, another in British-English. Not sure if there's one that's preferred here.

Reply:
We tend to follow American spellings, I believe.